CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 2.35.0-rc0 / 2022-04-06
+
+* [CHANGE] Files *.tmp WAL are deleted when Prometheus is starting. #10317
+* [FEATURE] ServiceDiscovery have now a plugin system. #10495
```

Comment:
Hmmmm... as far as I can see this is not directly a user-visible feature, at least not in the sense that the binary that we release now has a user-visible plugin system. If I understand correctly, this is only to make it easier for people building Prometheus themselves to include/exclude SD methods. @roidelapluie What do you think about how to communicate this?

Reply:
Yeah indeed, but I was thinking since it's a nice changes in the Prometheus ecosystem, perhaps it will be interesting to highlight it in the release.

Reply:
By our current changelog standards, we wouldn't mention it at all if it's not a direct user-visible change in the binaries we distribute. I think it's interesting in general for developers/builders (maybe as something to blog about it), but maybe not as part of these release notes?

Reply:
ah ok. In that case, yeah it's probably better to talk about it in a blog post or in a format @roidelapluie will find appropriate.

I removed it